php/tests/proto/test_wrapper_type_setters.proto

```diff
@@ -24,3 +24,14 @@ message TestWrapperSetters {
 
   map<string, google.protobuf.StringValue> map_string_value = 13;
 }
+
+message TestWrapperAccessorConflicts {
+  int32 normal_vs_wrapper_value = 1;
+  google.protobuf.Int32Value normal_vs_wrapper = 2;
```

Comment:
Appending the field number works for this example, but what if there was also a field named `normal_vs_wrapper_value_2`? I'm not suggesting that that is very likely, but WDYT about a more general solution to this problem that would guarantee no conflicts. For example, inside a class, create a container of names used for methods, and each time you generate a method, add it to the container, check for conflicts, and if there is a conflict, resolve it by appending an incrementing counter - keep incrementing until the name is unique. This would also cover any other possible cases of collision (I'm not sure if any other cases exist in PHP).

Reply:
This will forces us to delay generating all accessors until we have traversed all fields of the same message. I feel this will be quite complicate.
For now, we used the same approach in java (which also has the problem of conflict after renaming)
In future, we would like to introducing some rules for naming a field, like no field can be named as a prefix of another field.

Reply:
Ok, sgtm.